frontend/components/pages/volunteer/index.tsx

```diff
@@ -0,0 +1,37 @@
+import Link from "next/link";
+import React from "react";
+import { IFaq } from "types/faq";
+import { IVolunteer } from "types/volunteer";
+import { FAQS } from "./faq";
+
+export const VolunteerPageComponent: React.FC<{ volunteerPageContent: IVolunteer[]; faqs: IFaq[] }> = ({
+  volunteerPageContent,
+  faqs,
+}) => {
+  return (
+    <>
+      {volunteerPageContent.map((volunteer) => (
```

Comment:
Interesting approach, I hadn't thought about having the volunteer page content as its own database endpoint but actually it allows us to create multiple page sections (as we loop through each entry which could have some content) so I think that should work well